rest-api-spec/src/main/resources/rest-api-spec/test/suggest/30_context.yml

```diff
@@ -240,9 +237,7 @@ setup:
       indices.refresh: {}
 
   - do:
-      indices.get_mapping:
-        include_type_name: false
-
+      indices.get_mapping: {}
```

Comment:
I think an extra new line got deleted here?

Reply:
👍 will add it back.